### PR TITLE
BIND: Add version 9.14.3

### PR DESCRIPTION
**Pull request by linsui**
`dig` has been in scoop and it's a part of `bind` . Should I remove `dig.json` ? There is a `vcredist` in dir, could use it or must use `suggest` ?

### CODE REVIEW

**Comment by linsui** [comment]
@rasa Should I remove `dig.json` as it's just a part of bind?

**Comment by rasa** [comment]
> @rasa Should I remove `dig.json` as it's just a part of bind?

Removing it disrupts users. Perhaps we rewrite it to simply depend on bind, and add a note that it’s deprecated, and will be removed in the future?

Perhaps something like (untested)?:

```json
{
    "homepage": "https://www.isc.org/",
    "description": "dig (domain information groper) is a flexible tool for interrogating DNS name servers",
    "license": "MPL-2.0",
    "version": "9.14.3",
    "checkver": {
        "url": "https://www.isc.org/downloads/",
        "re": "Current-Stable[\\S\\s]*?BIND(.+)\\.x64\\.zip\""
    },
    "autoupdate": {
        "url": "https://example.com/"
    },
    "depends": "bind",
    "url": "https://example.com/",
    "hash": "",
    "notes": [
        "The dig package has been deprecated, and will be removed in the future.",
        "Please use the bind package instead."
    ]
}

**Comment by linsui** [comment]
Good idea. 👍 I'll test it.

File: bucket/dig.json [hunk 1]
```diff
@@ -1,34 +1,12 @@
...
-            "32bit": {
-                "url": "https://ftp.isc.org/isc/bind$majorVersion/$version/BIND$version.x86.zip"
-            }
-        }
-    },
-    "suggest": {
-        "vcredist": "extras/vcredist2012"
-    }
+    "depends": "bind",
+    "url":"https://www.isc.org/bind/",
```
Comment (Ash258) [inline bucket/dig.json]: I do not understand.

Instead of keeping funtional manifest and adding notes you will break whole manifest.

Comment (rasa) [reply bucket/dig.json]: Nothing broke. I tested it locally, and it worked fine. What do you think is broke? Is there a better way to do this? Should we just leave the dig package alone, and let there be two possible digs, if they install both bind and dig?

Comment (linsui) [reply bucket/dig.json]: It works fine for me. If you have suggestions, please tell me.

Comment (Ash258) [reply bucket/dig.json]: This manifest is completely useless now.
There is blank URL with just webpage. That does not make sense. Remove whole manifest or keep it working as berfore and add notes about bind

Comment (linsui) [reply bucket/dig.json]: I think it's acceptable as a temporary solution. But I find another problem. Scoop won't check `depend` when update so `bind` won't be installed automatically when update dig. Is it a bug?

**Comment by linsui** [comment]
We really need some way to migrate to a new package painless or just show a note when manifest is removed.

File: bucket/dig.json (hunk 1 above)
Comment (rasa) [reply bucket/dig.json]: @Ash258 The URL is required (at least according to the [wiki](https://github.com/lukesampson/scoop/wiki/App-Manifests#required-properties), but not the [schema](https://github.com/lukesampson/scoop/blob/b9b48182b4d921aa92e9b3e53ab2d2c376d467d7/schema.json#L526)) but is unneeded in this context, as we just are installing the `depends app. Perhaps it should be clear that that’s the case:
```
"url": "https://example.com/#url-parameter-required-but-is-not-needed-for-this-metapackage”
```
Or is the URL param no longer needed?

Comment (rasa) [reply bucket/dig.json]: @Ash258 Nevermind. @linsui has implemented your suggestion, which is fine.

Comment (Ash258) [reply bucket/dig.json]: It is needed.

But what is wrong with fully working `dig.json` with just notes about adopting to BIND??
Keeping manifest which does not install anything is completely nonsense in official bucket.

Comment (rasa) [reply bucket/dig.json]: @Ash258 Your proposal is a fine solution, but my proposed dig manifest installed dig by depending on bind, which has dig. In fact, both dig and bind now install the exact same download. The only difference is dig exposes only dig.exe via a shim, and bind exposes all ~15 .exes in the download via an `env_add_path`. Sorry for the confusion.

**Pull request merged**